src/components/shared/refreshBar/refreshBar.container.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { connect } from 'react-redux';
+import { translate } from 'react-i18next';
```

Comment:
Do you need translate?  If Header was relying on the t() function before, where was it getting it from? Do those locations need to be cleaned up?

Reply:
Also note, with my earlier checkin... we won't use translate() anymore, instead use withNamespaces(). Its the same thing, they just renamed it in a later version.

Reply:
Looks like translate is still there.... and named translate instead of withNamespaces. Is it needed here?